maintainers/maintainer-list.nix

```
@@ -2637,10 +2637,13 @@
     name = "Roman Gerasimenko";
   };
   builditluc = {
-    email = "builditluc@icloud.com";
+    email = "git@builditluc.eu";
     github = "Builditluc";
     githubId = 37375448;
-    name = "Buildit";
+    name = "Builditluc";
+    keys = [{
+      fingerprint = "FF16E475723B8C1E57A6B2569374074AE2D6F20E";
+    }];
```

The nixfmt check fails and looking at other entries it seems this should be formatted as follows:
```suggestion
    keys = [ { fingerprint = "FF16E475723B8C1E57A6B2569374074AE2D6F20E"; } ];
```